### ISSUE TITLE
[Feature] add cli tool like xonfig web

### ISSUE THREAD

**Issue by rapha8l** [xonsh/xonsh#3877]
Hi

Thanks for xonsh which is great
As I often use servers without gui, configuring xonsh is not as easy as having a desktop with a webbrowser
It would be great to have a text interface like xonfig web
Thanks

Raphaël

**Comment by anki-code** [comment]
Hi @rapha8l!
Have you tried `xonfig wizard`? If yes what in `xonfig web` helps to you in most?

You can find much more in `xonfig --help`:
```
$ xonfig --help
usage: xonfig [-h] {info,web,wizard,styles,colors,tutorial,jupyter-kernel} ...

Manages xonsh configuration.

optional arguments:
  -h, --help            show this help message and exit

action:
  {info,web,wizard,styles,colors,tutorial,jupyter-kernel}
    info                displays configuration information, default action
    web                 Launch configurator in browser.
    wizard              Launch configurator in terminal
    styles              prints available xonsh color styles
    colors              preview color style
    tutorial            Launch tutorial in browser.
    jupyter-kernel      Generate xonsh kernel for jupyter.

```

**Comment by rapha8l** [comment]
Thanks
In fact I missed the styles and colors options, though they are not interactive and you can't set the configuration with them

**Comment by scopatz** [comment]
Yeah, this tool needs a bit of an overhaul and tightening down to behave more like the web config.  PRs very welcome in this space!

**Comment by rapha8l** [comment]
I'd love to, though I hardly can do a hello world in python :)

**Comment by anki-code** [comment]
Why you can't just copy xonshrc to the server instead of using wizard? :)

**Comment by scopatz** [comment]
Well, I think you could in most cases, but it would be good to have a CLI config tool too

**Comment by rapha8l** [comment]
Indeed I copy my conf to my servers, but it's not very convenient, and I thought it could be useful for others too

**Comment by rapha8l** [comment]
or maybe have a version that works in a text browser like lynx or w3m

**Comment by scopatz** [comment]
> or maybe have a version that works in a text browser like lynx or w3m

haha wow, that is an amazing idea!

**Comment by rapha8l** [comment]
thanks :)
